pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+### Description
+<!--
+_Using one or more sentences, describe the proposed changes and the reason for making them._
+-->
+
+### Related JIRA Issue(s)
+<!--_Please provide the URL(s) for any JIRA issues related to this PR._-->
+
+### Review App URL(s) 
+http://__BRANCH_NAME_HERE__.review.ensembl.org
+
+### Knowledge Base
+<!--
+_If the PR introduces new concept, then provide link(s) to the Knowledge base ( Documentation, Blog etc )_
+-->
+
+### Example(s)
+<!--
+_Any details that will help reviewers to review the PR, such as  (but not limited to ) expected request/response, instruction for viewing the changes via the web client, or any other relevant information._
+-->
+
+### Checklist
```

Comment:
If you add checkboxes to the PR template, they will look like a TODO list in the github interface. This may not be something that you want, because TODOs sometimes are valuable on their own. We had such checkboxes in ensembl-client PR template, and then got rid of them.